spec/WideDemographicData.csvschema.json

```diff
@@ -0,0 +1,12 @@
+{
+    "columns": {
+        "country_code_numeric": "Int",
```

Comment:
Why `country_code_numeric`? Is this a thing in the existing long format?

Reply:
This was added to the spec here: https://vimc.myjetbrains.com/youtrack/issue/VIMC-753

Reply:
:+1: